frontend/src/components/common/StudentDetailCommon.vue

```diff
@@ -602,22 +652,69 @@ export default {
         this.genderHint
       ];
     },
-    validateMincode() {
+    validateMinCode() {
```

Comment:
Could we use the approach described in the vuetify document to define validateMinCode validation rule?